Release 0.2.2

istreeter opened the pull request.

alexanderdean commented on CHANGELOG:
```diff
@@ -1,3 +1,20 @@
+Bump AWS SDK to 1.11.208 (#26)
+Make SparkContext initializing eagerly (#24)
+
+
```
Should be one line between releases

alexanderdean commented on CHANGELOG:
```diff
@@ -1,3 +1,20 @@
+Version 0.3.0 (2017-11-XX)
```
Fix date as 28

istreeter committed:
Use TIMESTAMP_TZ types when creating the table (#36)

Previously we used the `TIMESTAMP` column type. In Snowflake, this is
_probably_ an alias for `TIMESTAMP_NTZ`, but actually it depends on
whether the account owner has change the default of the
`TIMESTAMP_TYPE_MAPPING` session parameter.

We should pick a concrete timestamp type and use it explicitly for all
new tables. This will prevent future confusion where we are not 100%
sure what timestamp got used when creating the table.

This change was supposed to happen in the first release, but it got
overlooked.

istreeter committed:
Missing atomic column should be treated as a setup error (#38)

Before this PR, the loader would crash immediately if the events table
is missing a required column like `load_tstamp`. It is better if the
loader stays alive long enough to send an informative message to the
monitoring webhook.

istreeter committed:
Treat all client errors as setup errors when opening channel (#37)

Snowflake responds with a 4xx response code for various reasons when
opening a channel:

- User does not have permissions (403)
- This table type does not support channels (400)
- There are probably others I have not seen yet!

Before this PR, if we treated the 403 as a setup error, but actually all
4xxs are probably setup errors. We should treat them gracefully by
staying alive and sending an informative message to the monitoring
webhook.

istreeter committed:
Prepare for 0.2.2 release
